src/gtk/mimetype.cpp

```diff
@@ -0,0 +1,54 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/gtk/mimetype.cpp
+// Purpose:     classes and functions to manage MIME types
+// Author:      Hans Mackowiak
+// Modified by:
+// Created:     05.06.2016
+// Copyright:   (c) 2016 Hans Mackowiak <hanmac@gmx.de>
+// Licence:     wxWindows licence (part of wxExtra library)
+/////////////////////////////////////////////////////////////////////////////
+
+
+#include "wx/wxprec.h"
+
+#ifdef __BORLANDC__
+    #pragma hdrstop
+#endif
+
+#if wxUSE_MIMETYPE && wxUSE_FILE
+
+#include "wx/gtk/mimetype.h"
+
+#include <gio/gio.h>
```

Comment:
Can we be sure that it's always available? Or could we need `PKG_CHECK_MODULES(gio)` in configure?